net/cryfs/Portfile

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem                1.0
+PortGroup                 github 1.0
+PortGroup                 cmake 1.1
+PortGroup                 active_variants 1.1
+
+github.setup              cryfs cryfs 0.10.2
+github.tarball_from       releases
+
+license                   LGPL-3
+
+categories                net fuse
+platforms                 darwin
+
+maintainers               @jolan78 openmaintainer
+
+description               Cryptographic filesystem for the cloud
+long_description          CryFS encrypts your files, so you can safely store them anywhere.\
+                          It works well together with cloud services like Dropbox, iCloud,\
+                          OneDrive and others
+
+homepage                  https://www.cryfs.org/
+
+checksums                 rmd160  9abdb5e59263cf8ea978470f4c6a4086ddd8a860\
+                          sha256  004c7fdf537062e197aaa0bcdafd2682904a279802441bb6063cb72c1fba52d2\
+                          size    10187912
+
+extract.mkdir             yes
+
+depends_build             port:openssl\
+                          port:python36\
+                          port:boost\
+                          path:/usr/local/include/osxfuse/fuse.h:osxfuse
+
+depends_lib               port:curl\
+                          port:libomp\
+                          path:/usr/local/lib/libosxfuse.2.dylib:osxfuse
+
+cmake.build_type          Release
+universal_variant         no
+
+configure.args-append    -DBoost_USE_MULTITHREADED=on
+
+variant no_version_check description {Doesn't check online for updates and security vulnerabilities} {
+    configure.args-append -DCRYFS_UPDATE_CHECKS=off
+}
```

Comment:
This should not be a variant, you should always do this and keep the Portfile updated.

Reply:
this variant is for cryfs to display a message when you mount a volume if the upstream version changed. I assume you thought it was a build-time check ?
Note that if we get rid of this variant, we could let update checking enabled (which is the default) and let the user define the environment variable CRYFS_NO_UPDATE_CHECK which disables this behavior. I still think preventing accidental update check is a good idea but you have the final word.

Reply:
I think this might make sense as a variant given the description.

Reply:
@mf2k : what is your advice on this variant ?

Reply:
I think mf2k wants the variant (not doing the version check) to be the default. How about we invert it and provide a variant (which isn't a default) that does the check, so the installed default does not?